app/Http/Controllers/UserController.php

```diff
@@ -2,18 +2,90 @@
 
 namespace App\Http\Controllers;
 
-use Illuminate\Http\Request;
+use App\Http\Requests\UserStore;
+use App\Http\Requests\UserUpdate;
+use Illuminate\Support\Facades\DB;
 use Illuminate\Support\Facades\Gate;
+use Illuminate\Support\Facades\Hash;
 
 class UserController extends Controller
 {
-    public function __invoke(\App\User $user)
+    public function show()
+    {
+        return response(auth()->user());
+    }
+
+    public function store(UserStore $request)
+    {
+        DB::beginTransaction();
+
+        try {
+            $validated = $request->validated();
+            $user = \App\User::create($validated);
+            $user->phones()->createMany($validated['phones']);
+            $user->addresses()->createMany($validated['addresses']);
```

Comment:
Como a gente vai fazer outra tela para cadastrar o endereço do usuário, você pode criar um controller separado para o endereço.

Reply:
No início o usuário apenas vai cadastrar um endereço então não precisa do createMany... a mesma coisa para o telefone.

Reply:
Certo, mas qual o porquê de uma administrador precisar de múltiplos telefones quando o usuário não precisa?